src/components/bcros/filing/item/DissolutionVoluntary.vue

```diff
@@ -46,20 +39,7 @@ const dissolutionDateTime =
       <div v-else-if="isStatusCompleted" data-cy="completed-dissolution-details">
         <strong>{{ $t('text.filing.dissolution.completed') }}</strong>
 
-        <p v-if="isEntityFirm">
```

Comment:
we want to keep this v-if as different text is displayed based on whether the business is an entity firm or not. I think we get incorrect text in the screenshot because `isEntityFirm` is actually a function. Changing it to `v-if="isEntityFirm()"` can solve the issue.

Reply:
added back along with all the other stuff that was ripped out as part of removing